DeploymentExperiences.md

```diff
@@ -0,0 +1 @@
+* Got struck with settings for database and setting config variables on Heroku
```

Comment:
Heroku can be tricky to configure. I know you were having issues with an application error earlier in the week. If you've yet to fix this, let us know and we can try to resolve it. If you were able to fix it, great job! Struggling through these things sometimes is the best way to learn them.